README.md

```diff
@@ -150,3 +164,19 @@ limitations under the License.
   * A. No. Ubuntu 22.04 only for now.
 * Q. Is ROS 2 Foxy/Galactic supported?
   * A. No. ROS 2 Humble only for now.
+* Q. `colcon build` shows `1 package had stderr output: mini_pupper_control`.
```

Comment:
I believe it should be fixed by downgrading setuptools: `pip3 install setuptools==58.2.0`

Reply:
Thank you for the detailed information!
I would just like to point out that this error is an issue with the build tool and not with this ROS package.

Reply:
related issue: https://github.com/colcon/colcon-core/issues/454